control-plane/CMakeLists.txt

```diff
@@ -93,22 +93,28 @@ set (CONTROLPLANE_HDRS
   p4RuntimeTypes.h
   typeSpecConverter.h
   bfruntime.h
-  )
+)
 
 add_library(controlplane-gen OBJECT ${P4RUNTIME_GEN_SRCS})
-target_include_directories(controlplane-gen
-  SYSTEM BEFORE PUBLIC ${Protobuf_INCLUDE_DIRS}
+target_include_directories(
+  controlplane-gen SYSTEM BEFORE PUBLIC ${Protobuf_INCLUDE_DIRS} SYSTEM BEFORE
+  PUBLIC ${ABSL_COMMON_INCLUDE_DIRS}
```

Comment:
Probably worth a fixme, that if we'd include proper protobuf targets, then we can get rid of `Protobuf_INCLUDE_DIRS` as include dirs will be exposed via interface of protobuf link target